draft-rha-jose-hpke-encrypt.md

```diff
@@ -173,7 +173,7 @@ We provide the following table for additional clarity:
 | Name                   | Recipients | Serializations | Content Encryption Key | Similar to
 |---
 | Integrated Encryption  | 1          | Compact        | Derived from HPKE      | Direct Key Agreement
```

Comment:
```suggestion
| Integrated Encryption  | 1          | Compact, JSON        | Derived from HPKE      | Direct Key Agreement
```

Reply:
Are empty string "iv" and "tag" allowed?

Reply:
Previous thread: https://github.com/tireddy2/JOSE_HPKE/pull/14#discussion_r1485568237

Reply:
MUST be present when non-empty, otherwise must be absent from the JSON.

Reply:
[Thanks, ](https://github.com/tireddy2/JOSE_HPKE/pull/19/commits/1de1ad2af026caa99085abbaec0db79995681416)